index.html

```diff
@@ -10467,6 +10484,44 @@ <h3>Relationship Attributes</h3>
 		<h2>Definitions of States and Properties (all aria-* attributes)</h2>
 		<p>Below is an alphabetical list of <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> <a>states</a> and <a>properties</a> to be used by rich internet application authors. A detailed definition of each <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> state and <a>property</a> follows this compact list.</p>
 		<p id="index_state_prop" class="placeholder">Placeholder for index of states and properties</p>
+		<div class="property" id="aria-actions">
+			<pdef>aria-actions</pdef>
+			<div class="property-description">
+				<p><a>Identifies</a> a related element or elements whose primary activation (<code>click</code> event) will trigger behaviors or operations relevant to the current element.</p>
```

Comment:
I acknowledge "current element" could be ambiguous, but I'm not sure "object" improves the clarity. How about the following instead?
```suggestion
				<p><a>Identifies</a> a related element or elements whose primary activation (<code>click</code> event) will trigger a behavior or operation relevant to original item, such as close button for a dialog, or a reply button relevant to the specific email message in focus.</p>
```

Reply:
While "original item" is somewhat less confusing, it is just as inconsistent with every other state or property definition in the spec. Looking across all the state and property definitions, "the object" is consistently used as shorthand for "the object on which the attribute was specified". That is why I think "the object" is more clear -- it is how we say the exact same thing elsewhere.

Reply:
Hmm… I’ll defer to @smhigley or the chairs on this. 

My feeling is that “object” is one of the least specific words we could use. It can be useful to describe abstract concepts, but adds unnecessary ambiguity in a specific case like this.

Reply:
Conceding the point of using "the original user interface object" (rather than just "the object") which I think aligns with @mcking65's feedback and is specific enough here to satisfy my objection to the proposed language.
```suggestion
				<p><a>Identifies</a> a related element or elements whose primary activation (<code>click</code> event) will trigger a behavior or operation relevant to the original user interface object, such as the close button for a dialog, or a reply button relevant to the specific email message in focus.</p>
```

Reply:
@mcking65 agreed offline to "referencing user interface object." Agreed and committing to PR.

```suggestion
				<p><a>Identifies</a> a related element or elements whose primary activation (<code>click</code> event) will trigger a behavior or operation relevant to the referencing user interface object, such as the close button for a dialog, or a reply button relevant to the specific email message in focus.</p>
```